app/assets/javascripts/practice_editor_risk_mitigation.es6

```diff
@@ -90,12 +88,33 @@
         }
     }
 
+    function attachDeleteEntryHandler(elem) {
+        $(elem).on('click', (e) => {
+            let $previousLi = $(e.target).closest('.practice-editor-risk-mitigation-li').filter(":visible").prev('.practice-editor-risk-mitigation-li').filter(":visible");
+
+            let $afterLi = $(e.target).closest('.practice-editor-risk-mitigation-li').filter(":visible").next('.practice-editor-risk-mitigation-li').filter(":visible");
+
+            let $allLi = $('.practice-editor-risk-mitigation-li').filter(":visible")
+
+            // remove previous element separator if there is no element after the one being deleted
+            if ($previousLi.length > 0 && $afterLi.length === 0) {
+                $previousLi.find('.add-another-separator').filter(":visible").remove();
+            }
+
+            // if there are no more <li> elements display the add button
+            if ($allLi.length <= 1) {
+                $('#dm-add-button-risk-mitigation').removeClass('display-none');
+                $('#dm-add-link-risk-mitigation').addClass('display-none');
+            }
+        })
+    }
+
     function loadPracticeEditorRiskMitiFunctions() {
         hideAddLinksAndShowRiskMitiFields();
         createUniqueRiskMitiId();
-        removeBulletPointFromNewLi();
-        dragAndDropRiskMitigationListItems();
+        attachDeleteEntryHandler('.risk-mitigation-trash');
+        // dragAndDropRiskMitigationListItems();
```

Comment:
Note: commented this out for now in case we need the drag and drop code for later... let me know if it's better to just delete this 😅